drizzlepac/alignimages.py

```diff
@@ -1000,29 +972,25 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, print_fit
             fit_quality = 4
 
     if print_fit_parameters:
-        for item in imglist:
-            log.info(fit_status_dict["{},{}".format(item.meta['name'], item.meta['chip'])])
+        for item in imglist: log.info(fit_status_dict["{},{}".format(item.meta['name'], item.meta['chip'])])
 
     if max_rms_val > MAX_FIT_RMS:
-        log.info(
-            "Total fit RMS value = {} mas greater than the "
-            "maximum threshold value {}.".format(max_rms_val, MAX_FIT_RMS))
+        log.info("Total fit RMS value = {} mas greater than the maximum threshold value {}.".format(max_rms_val, MAX_FIT_RMS))
     if not overall_valid:
-        log.info(
-            "The fit solution for some or all of the images is not valid.")
-    if max_rms_val > MAX_FIT_RMS or overall_valid is False:
+        log.info("The fit solution for some or all of the images is not valid.")
+    if max_rms_val > MAX_FIT_RMS or overall_valid == False:
```

Comment:
`overall_valid == False` should be rewritten as `overall_valid is False`

Reply:
Resolved in 51e3cea